src/components/button/_mixins.scss

```diff
@@ -81,7 +81,7 @@
   flex-shrink: 0;
   font-size: $button-font-size;
   font-weight: $button-font-weight;
-  height: auto;
+  height: rem(48px);
```

Comment:
IMO this should be `$button-height` -- see: https://github.com/IBM/carbon-components/blob/e4da4a57542a42f7c6e0a90e91aac9b7e0fcaf02/src/globals/scss/_theme-tokens.scss#L140